internal/compile/ucast.go

```diff
@@ -107,6 +107,10 @@ func toFieldNode(op string, r ast.Ref, v ast.Value, opts *Opts, refOK bool) *uca
 				Field: f,
 			}
 		}
+	case ast.Var:
```

Comment:
We know at this point that if we have a `ast.Var` type, we're not doing a comparison?